controllers/C_Document.class.php

```diff
@@ -686,7 +694,11 @@ function retrieve_action(string $patient_id = null, $document_id, $as_file = tru
                 // save the to-file if a to-file was created in above convert call
                 if (is_file($to_file_tmp_name)) {
                     $couchI = new CouchDB();
-                    $json = json_encode(base64_encode(file_get_contents($to_file_tmp_name)));
+                    if ($d->get_encrypted() == 1) {
+                        $json = json_encode($this->cryptoGen->encryptStandard(file_get_contents($to_file_tmp_name), null, 'database'));
+                    } else {
+                        $json = json_encode(base64_encode(file_get_contents($to_file_tmp_name)));
+                    }
                     $couchdata = array($GLOBALS['couchdb_dbase'], "converted_" . $couch_docid, $d->get_foreign_id(), "", "image/jpeg", $json);
```

Comment:
remove the json_encode wrapping here (needed to do this or else the data was getting double json'd